### PR TITLE
Add robupy and implement worst case value function iteration

### PR DESCRIPTION
**Pull request by MaxBlesch**
I use robupy to calculate the worst case probabilities and the worst case value function.

### CODE REVIEW

File: .envrc [hunk 1]
```diff
@@ -1,14 +1,25 @@
 source activate ruspy_analysis
 
-# We need access to ruspy submodule
-export PYTHONPATH=$PWD/submodules/ruspy
+
+# We need access to robupy submodule
+export PYTHONPATH=$PWD/submodules/robupy
```
Comment (peisenha) [inline .envrc:5]: Please create one block of edits to `PYTHONPATH` and then one applicable comment. All but one are wrong.

File: configurations/jupyter/migrated [hunk 2]
```diff
@@ -0,0 +1 @@
+2019-08-26T14:02:30.274047
```
Comment (peisenha) [inline configurations/jupyter/migrated]: I do not think this file is needed?!

File: python/execute_notebooks.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import subprocess as sp
+import glob
+import os
+
+os.chdir(os.environ['PROJECT_ROOT'] + '/notebooks')
+
+for notebook in sorted(glob.glob('*.ipynb')):
+    
+    # TODO: The others are not yet ready for this.
+    if notebook != '00_illustrations.ipynb':
```
Comment (peisenha) [inline python/execute_notebooks.py]: This only executes one of the notebooks not all of them, What is our Roadmap to including all?

File: notebooks/worst_case_probs.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    num_states, p_ml, costs, beta, rho, threshold=1e-8, max_it=100000000
+):
+    ev = np.zeros(num_states)
+    worst_trans_mat = trans_mat = create_transition_matrix(num_states, p_ml)
+    ev_new = np.dot(trans_mat, np.log(np.sum(np.exp(-costs), axis=1)))
+    while (np.max(np.abs(ev_new - ev)) > threshold) & (max_it != 0):
+        ev = ev_new
+        maint_value = beta * ev - costs[:, 0]
+        repl_value = beta * ev[0] - costs[0, 1] - costs[0, 0]
+        ev_min = maint_value[0]
```
Comment (peisenha) [inline notebooks/worst_case_probs.py:37]: This is the recentering in Equation 3.20 in Rust (200), please give this a proper name / comment. This threw me of a couple of times understanding the code.

**Review by peisenha — CHANGES_REQUESTED** [review]
- Please move all ifgures, i.e. *.png files, into our analysis/figures directory. Also, adjust codes so that if they are created they are put there. See the https://github.com/OptionValueHumanCapital/analysis/blob/master/notebooks/ for an example solution.

File: configurations/jupyter/migrated (hunk 2 above)
Comment (MaxBlesch) [reply configurations/jupyter/migrated]: Added it to .gitignore. It is automatically created.

File: python/execute_notebooks.py (hunk 3 above)
Comment (MaxBlesch) [reply python/execute_notebooks.py]: Added investigation notebook. Will add other notebooks later.

File: notebooks/worst_case_probs.py (hunk 4 above)
Comment (MaxBlesch) [reply notebooks/worst_case_probs.py:37]: Added comment.

File: .envrc (hunk 1 above)
Comment (MaxBlesch) [reply .envrc:5]: Adjusted .envrc. I think it doesn't look great. Maybe you have some more feedback. To cite you in another discussion: " Es gibt da schon ein Problembewusstsein :) "

**Review by peisenha — APPROVED** [review]

**Pull request merged**